vmdb/app/models/mixins/assignment_mixin.rb

```diff
@@ -85,11 +85,11 @@ def namespace
   module ClassMethods
     def assignments(find_options = {})
       # Get all assigned, enabled instances for type klass
-      ass_hash = self.all(find_options).inject({}) {|h,a| h[a.id] = a; h}
-      tags = Tag.all(:conditions => ["taggings.taggable_type = ? and tags.name like ?", self.name, "#{self.namespace}/%"], :include => "taggings")
+      assignment_map = self.all(find_options).inject({}) { |h, a| h[a.id] = a; h }
```

Comment:
- Should this be `all(find_options)` or `where(find_options)`
- The `.inject...` simplifies to `.index_by(&:id)`